herddb-core/src/main/java/herddb/cluster/BookKeeperDataStorageManager.java

```diff
@@ -1207,8 +1256,9 @@ private static LogSequenceNumber readLogSequenceNumberFromIndexMetadataFile(Stri
     @Override
     public List<Table> loadTables(LogSequenceNumber sequenceNumber, String tableSpace) throws DataStorageManagerException {
         try {
-            String tableSpaceDirectory = getTableSpaceZNode(tableSpace);
-            ensureZNodeDirectory(tableSpaceDirectory);
+//            TODO
```

Comment:
Impl leftovers?

Reply:
actually this is no more needed.
I am dropping this commented code, thanks